bin/web

```diff
@@ -55,7 +55,7 @@ dev() {
     esac
   done
 
-  cd "$ROOT"/web/app && yarn webpack-dev-server --port "$DEV_PORT" &
+  cd "$ROOT"/web/app && yarn webpack serve --port "$DEV_PORT" &
```

Comment:
Does this support live-reloading that the dev server provides?

Reply:
Yes, live reloading should be on by default if webpack is started with `NODE_ENV` set to `development`.